lib/ratyrate/helpers.rb

```diff
@@ -38,6 +38,9 @@ def rating_for(rateable_obj, dimension=nil, options={})
     else
       readonly = !current_user || false
     end
+    
+    #ReadOnly
+    readonly = options[:readonly] if options[:readonly].present?
```

Comment:
thanks for adding missing read only :) 

But, please can you change this to:

````
read_only = if options.has_key?(:read_only)
                        options[:read_only] == true
                    elsif disable_after_rate
                        !(current_user && rateable_obj.can_rate?(current_user, dimension))
                    else
                       false
                    end
````
`